src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/BootableVolumeList.tsx

```diff
@@ -187,9 +186,17 @@ const BootableVolumeList: FC<BootableVolumeListProps> = ({
           />
           <BootableVolumesPipelinesHint bootableVolumes={bootableVolumes} />
         </>
-      ) : (
-        <BootableVolumeEmptyState />
       )}
+      {!displayVolumes &&
```

Comment:
consider splitting `displayVolumes` into simpler  flags  and re-defining this condition . After expanding the condition looks confusing:
```
!isEmpty(bootableVolumes) && loaded && loadedColumns && (loaded ? <Empty> : <Loading>)
```